datatables.go

```diff
@@ -290,11 +295,60 @@ func (p *Parser) bindWeapons() {
 
 	for _, sc := range p.stParser.ServerClasses() {
 		for _, bc := range sc.BaseClasses {
-			if bc.Name == "CWeaponCSBase" {
+			switch bc.Name {
+			case "CWeaponCSBase":
 				sc.RegisterEntityCreatedHandler(p.bindWeapon)
+			case "CBaseGrenade": // Grenade that has been thrown by player.
+				sc.RegisterEntityCreatedHandler(p.bindGrenadeProjectiles)
+			case "CBaseCSGrenade":
+				// @micvbang TODO: handle grenades dropped by dead player.
```

Comment:
For the future or this PR? What's it about, I don't exactly understand

Reply:
For the future; it's for grenades that you can pick up from the ground, i.e. dropped by dead players